doc/usecases/cli/UC_get_config.md

```diff
@@ -0,0 +1,22 @@
+# Use Case: cli get configuration
+
+## Summary
+
+- **Scope:** CLI use
+- **Level:** Developer Goal, UI
+- **Actors:** CLI user
+- **Brief:** User reads data for a given keyname
+
+## Scenarios
+
+- **Precondition:** -
+- **Main success scenario:**
+  - Backend is asked for the data for the given keyname
+  - the data is outputted
+- **Alternative scenario:** -
+- **Error scenario:**
+  - The keyname does not exist
+  - The keyname is not valid
+  - The keyname contains a bookmark that does not exist
+- **Postcondition:**
+  - Data for keyname or error-message is outputted
```

Comment:
Error scenarios are usually not part of the postcondition (as other use cases written now). If you want to be more precise, split up postcondition to minimal guarantee and success guarantee. (See Writing effective Use cases).

```suggestion
  - Value of key is printed
```